Yamb/CustomUIElements/CustomLoginButton.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CustomLoginButton.swift
+//  Yamb
+//
+//  Created by Ana Peshevska on 5.7.22.
+//
+
+import UIKit
+
+class CustomLoginButton: UIButton {
+    init() {
+        super.init(frame: .zero)
+        self.backgroundColor = .systemBlue
+        self.setTitleColor(.white, for: .normal)
+        self.layer.cornerRadius = 5
```

Comment:
it's good practice to define the constants outside of the class. I would create a `let cornerRadius = 5` on line 9. this way if you have the corner radius in multiple places, you would only need to change it in one place.